drivers/bridge/bridge.go

```diff
@@ -76,15 +78,45 @@ func New() (string, driverapi.Driver) {
 	return networkType, &driver{}
 }
 
+func (d *driver) Config(option interface{}) error {
+	var config *Configuration
+
+	d.Lock()
+	defer d.Unlock()
+
+	if d.config != nil {
+		return fmt.Errorf("configuration already exists, simplebridge configuration can be applied only once")
```

Comment:
am not a big fan of type-less errors. But I understand that this is a general pattern across the project.
Have opened #53 to address this.